packages/amplify-e2e-tests/src/__tests__/auth_2.test.ts

```diff
@@ -1,12 +1,15 @@
 import {
-  addAuthViaAPIWithTrigger,
   addAuthWithDefaultSocial,
-  addAuthWithGroupTrigger,
-  addAuthWithRecaptchaTrigger,
-  addAuthwithUserPoolGroupsViaAPIWithTrigger,
   amplifyPull,
   amplifyPush,
   amplifyPushAuth,
+  addAuthWithGroupTrigger,
+  addAuthViaAPIWithTrigger,
+  addUserToUserPool,
+  // eslint-disable-next-line spellcheck/spell-checker
+  addAuthWithRecaptchaTrigger,
+  // eslint-disable-next-line spellcheck/spell-checker
```

Comment:
I'm in favor of adding "recaptcha" but less in favor of adding "authwith". If it's simple to rename that function we should do that, otherwise make it a named import and only ignore spellcheck at the top of the file rather than everywhere in the file

Reply:
I tried adding this ```recaptcha``` to dictionary but it was complaining anyways (not sure why).
Since this function is used once , named import also start throwing lint errors as named imports are unnecessary

For now I have added spell check for entire file